packages/amazonq/package.json

```
@@ -627,7 +627,7 @@
                 "title": "%AWS.command.codewhisperer.signout%",
                 "category": "%AWS.amazonq.title%",
                 "icon": "$(debug-disconnect)",
-                "enablement": "aws.codewhisperer.connected"
+                "enablement": "aws.codewhisperer.connected && !aws.amazonq.isSagemakerUnifiedStudio"
```

This context var is indicating an environment thing. It's not specific to Q. So the same name should be used in both Q and Toolkit:

```suggestion
                "enablement": "aws.codewhisperer.connected && !aws.isSagemakerUnifiedStudio"
```

---

Ack, thanks for the comment. Addressing in next rev